gemato/openpgp.py

```diff
@@ -383,7 +390,11 @@ def home(self):
         assert self._home is not None
         return self._home
 
+    def update_env(self, key, value):
+        if key in ENVIRON_WHITELIST:
+            self._env[key] = value
+
     def _spawn_gpg(self, options, stdin=''):
-        env_override = {'GNUPGHOME': self.home}
+        self._env['GNUPGHOME'] = self.home
```

Comment:
Actually, I suppose you could do that straight in `__init__` since `_home` is already defined there. Check if `home` is still used then.

Reply:
Yes indeed it's not used anymore. I've removed it and set it into `self._env` in `__init__`